packages/core/src/components/essentials/tooltip-axis.ts

```diff
@@ -84,6 +84,14 @@ export class AxisChartsTooltip extends Tooltip {
 					color: this.model.getStrokeColor(datum[groupMapsTo])
 				}
 			];
+
+			// Add radius label and value
+			if (e.detail.radiusLabel) {
+				items.push({
+					label: e.detail.radiusLabel,
+					value: datum[e.detail.radiusMapsTo] 
+				})
```

Comment:
seems like if `getItems()` supports an `additionalItems` array, then bubble can provide an additional radius value to it instead of tooltip knowing about bubble